Kiss.Bff/Intern/ContactmomentDetails/Features/ContactmomentDetailsDetails.cs

```diff
@@ -1,27 +1,27 @@
-﻿using System.Globalization;
-using Kiss.Bff.Beheer.Data;
-using Microsoft.AspNetCore.Authorization;
+﻿using Kiss.Bff.Beheer.Data;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.EntityFrameworkCore;
 
-namespace Kiss.Bff.ZaakGerichtWerken.Contactmomenten
+namespace Kiss.Bff.Intern.ContactmomentDetails.Features
 {
     [ApiController]
     public class ReadContactmomentDetails : ControllerBase
     {
         private readonly BeheerDbContext _db;
-        private const int MaxPageSize = 5000;
 
         public ReadContactmomentDetails(BeheerDbContext db)
         {
             _db = db;
         }
 
-        [HttpGet("/api/contactmomentdetails")]      
+        [HttpGet("/api/contactmomentdetails")]
         public async Task<IActionResult> Get([FromQuery] string id, CancellationToken token)
         {
             var contactmoment = await _db.ContactMomentDetails
-                .FirstOrDefaultAsync(x => x.Id == id);
+                .Where(x => x.Id == id)
```

Comment:
vanwaar deze aanpassing in dit endpoint?

Reply:
ik heb de code naar de juiste plek verplaatst en een scheiding gemaakt tussen model en entity. dit is meer in lijn met de rest van de applicatie en zorgt ervoor dat we geen navigation properties meesturen naar de client.

Reply:
Mee eens, aangepast